cfme/automate/service_dialogs.py

```diff
@@ -129,6 +129,8 @@ def create(self, *element_data):
                 version.LOWEST: "Add a New Element to this Box",
                 "5.3": "Add a new Element to this Box"})
             sel.wait_for_element(element_form.ele_label)
+            fill(element_form, {'choose_type': "Text Area Box"})
+            fill(element_form, {'choose_type': "Check Box"})
```

Comment:
I don't understand what this is for? Is there a bug in the widgets selector function? We should try to fix that first as opposed to shimming.